pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.black]
+line-length = 88
+include = '\.pyi?$'
```

Comment:
These options are all the default `black` options, do we need to specify them again?